code/game/objects/machinery/vending/new_marine_vendors.dm

```diff
@@ -1081,7 +1081,9 @@ GLOBAL_LIST_INIT(marine_selector_cats, list(
 	req_access = list(ACCESS_MARINE_SMARTPREP)
 
 	listed_products = list(
-		/obj/item/storage/box/t26_system = list(CAT_ESS, "Essential Smartgunner Set", 0, "white"),
+		/obj/item/storage/box/t26_system = list(CAT_ESS, "T-26 Set", 0, "white"),
+		/obj/item/storage/box/t26_system/t25 = list(CAT_ESS, "T-25 Set", 0, "white"),
```

Comment:
Can you add something that can make me differentiate these apart from a literal number

Reply:
ok